runtime/lua/vim/treesitter/highlighter.lua

```diff
@@ -54,7 +54,7 @@ TSHighlighter.hl_map = {
     ["include"] = "Include",
 }
 
-function TSHighlighter.new(query, bufnr, ft)
```

Comment:
Note that this will be changed in the followup PR, to remove the `bufnr`/ `ft`use and use parsers instead.